Geometry/CommonTopologies/interface/SimplePixelTopology.h

```diff
@@ -230,35 +235,9 @@ namespace phase2PixelTopology {
       4,  6,  5,  7,  6,  8,  7,  9,  8,  10, 9,  11, 10, 12,  // POS Jump (48)
       16, 18, 17, 19, 18, 20, 19, 21, 20, 22, 21, 23, 22, 24,  // NEG Jump (55)
   };
-  HOST_DEVICE_CONSTANT uint32_t layerStart[numberOfLayers + 1] = {0,
-                                                                  108,
-                                                                  324,
-                                                                  504,  // Barrel
-                                                                  756,
-                                                                  864,
-                                                                  972,
-                                                                  1080,
-                                                                  1188,
-                                                                  1296,
-                                                                  1404,
-                                                                  1512,
-                                                                  1620,
-                                                                  1796,
-                                                                  1972,
-                                                                  2148,  // Fp
-                                                                  2324,
-                                                                  2432,
-                                                                  2540,
-                                                                  2648,
-                                                                  2756,
-                                                                  2864,
-                                                                  2972,
-                                                                  3080,
-                                                                  3188,
-                                                                  3364,
-                                                                  3540,
-                                                                  3716,  // Np
-                                                                  numberOfModules};
+  HOST_DEVICE_CONSTANT uint32_t layerStart[numberOfLayers + 1] = {
+      0,    216,  432,  612,  864,  972,  1080, 1188, 1296, 1404, 1512, 1620, 1728, 1904,           2080,
+      2256, 2432, 2540, 2648, 2756, 2864, 2972, 3080, 3188, 3296, 3472, 3648, 3824, numberOfModules};
```

Comment:
These are actually different values.
Is it intended ?
What is the reason for the change ?

Reply:
The module count changed with `D110` geometry (originally we spotted it in https://github.com/cms-sw/cmssw/issues/45177). Then with https://github.com/cms-sw/cmssw/pull/45421 I made this obsolete but I didn't see it was still used with to calculate the [number of layers](https://github.com/cms-sw/cmssw/blob/master/Geometry/CommonTopologies/interface/SimplePixelTopology.h#L66) (so it's a bug). With this PR it's not more used at all but I have changed it in order to have a fair comparison with Phase2 (that is how I noticed it). 

Shall I put it in a different PR?

Reply:
As it is a fix, and this PR is hopefully close to being ready, I agree to keep the fix here :-)

Reply:
Minor, but yet important comment, IMHO: I really liked the comments related to the transition between detectors.

Reply:
Sure, let me restore them.